test/integration/functional_test.go

```diff
@@ -399,8 +400,6 @@ func validateListImages(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
-// check functionality of minikube after evaling docker-env
-// TODO: Add validatePodmanEnv for crio runtime: #10231
 func validateDockerEnv(ctx context.Context, t *testing.T, profile string) {
```

Comment:
bring comment back for valdiateDockerEnv (only remove TODO)

Reply:
done